internal/npm_install/test/golden/@angular/core/BUILD.bazel.golden

```diff
@@ -677,7 +677,7 @@ node_module_library(
 node_module_library(
     name = "core__contents",
     srcs = [":core__files"],
-    scripts = [
+    named_module_srcs = [
```

Comment:
kinda breaking for users, huh?
maybe we can paste a buildozer script into the release notes?

Reply:
I don't think anyone is using node_module_library directly as it is still private API and their generated build files will be re-generated on update